<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentsPage</class>
 <widget class="QWidget" name="DocumentsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Document Hub</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="topWidget" native="true">
      <layout class="QHBoxLayout" name="topRowLayout" stretch="1,2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="topLeftLayout">
         <item>
          <widget class="QLabel" name="specDocumentsHeaderLabel">
           <property name="text">
            <string>Project Specifications</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="specDocumentsListWidget">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="topRightLayout">
         <item>
          <widget class="QLabel" name="reviewLogHeaderLabel">
           <property name="text">
            <string>Review Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="reviewLogBrowser"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bottomWidget" native="true">
      <layout class="QHBoxLayout" name="bottomRowLayout" stretch="1,2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="bottomLeftLayout">
         <item>
          <widget class="QLabel" name="otherDocumentsHeaderLabel">
           <property name="text">
            <string>Other Documents</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="otherDocumentsListWidget">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="reviewLogContentsWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="newLogEntryContentsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="newLogEntryHeaderLabel">
               <property name="text">
                <string>Add New Log Entry</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="authorLabel">
                 <property name="text">
                  <string>Author:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="authorComboBox">
                 <item>
                  <property name="text">
                   <string>DEVELOPER</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>CLIENT</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="logTextEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonRowLayout" stretch="1,2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="uploadVersionButton">
         <property name="text">
          <string>Upload New Version...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addOtherDocumentButton">
         <property name="text">
          <string>Add Other Document...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveFeedbackButton">
         <property name="text">
          <string>Save Feedback</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="markApprovedButton">
         <property name="text">
          <string>Mark as Approved</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomNavLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string> &lt; Back </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>